CMakeLists.txt

```diff
@@ -468,6 +468,14 @@ if(MSVC)
     add_definitions(-D_CRT_NONSTDC_NO_DEPRECATE)
 endif()
 
+# Check whether compiler supports -fno-semantic-interposition parameter
```

Comment:
Perhaps this block should be moved near check for `attribute(hidden)` in cmake?

Reply:
That was what I initially wanted, but I thought it depended on check_c_source_compile_or_run, now that I have finished it I see that it is not needed. Will rewrite to use check_c_source_compiles() instead, and move it up.

Reply:
Fixed